onboarding agb apb

nbittich opened the pull request:
OP-2239
Could be merged to dev but we could also wait till the final version of the data is ready

claire-lovisa commented on app/components/administrative-unit-select-by-name.js:
```diff
@@ -19,8 +19,17 @@ export default class AdministrativeUnitSelectByNameComponent extends Component {
         `;
-    } else {
+    }
+    // else if (this.currentSession.hasAgbApbRole) {
+    //   filter['classification_id'] = `
+    //      ${CLASSIFICATION.AGB.id},
+    //      ${CLASSIFICATION.APB.id},
+    //     `;
+    // }
```
Can this go ? :)

claire-lovisa commented on app/routes/administrative-units/index.js:
```diff
@@ -45,8 +45,17 @@ export default class AdministrativeUnitsIndexRoute extends Route {
         `;
-      } else {
+      }
+      // else if (this.currentSession.hasAgbApbRole) {
+      //   filter['classification_id'] = `
+      //    ${CLASSIFICATION.AGB.id},
+      //    ${CLASSIFICATION.APB.id},
+      //   `;
+      // }
```
Can this go too ? :p

claire-lovisa commented on app/services/current-session.js:
```diff
@@ -7,6 +7,8 @@ const EDITOR_ROLES = [
   'ABBOrganisatiePortaalErediensten-editeerder',
   'ABBOrganisatiePortaalGebruiker-beheerder',
   'ABBOrganisatiePortaalErediensten-beheerder',
+  //'ABBOrganisatiePortaalAgbApb-beheerder',
+  // 'ABBOrganisatiePortaalAgbApb-editeerder',
```
And this too ?

claire-lovisa commented on app/services/current-session.js:
```diff
@@ -57,6 +66,10 @@ export default class CurrentSessionService extends Service {
     return WORSHIP_ROLES.includes(this.role.activeRole);
   }
 
+  // get hasAgbApbRole() {
+  //   return AGB_APB_ROLES.includes(this.role.activeRole);
+  // }
```
And this

claire-lovisa reviewed (commented):
- When logging in via mock login, we can chose a agb/apb version, but from the frontend code I'd suggest that it's not relevant anymore, can we remove these mock-login accounts ?
- I don't see the new fields mentionned in the FP (Naam volgens KBO, verkorte naam, naam volgens statuten, oprichtingsdatum)
- I only one type of site, sems to be mising "correspondentieadres", and the site type doesn't seem to be set by default to "maatschappelijke zetel" when creating a new bestuur
- "Organisatie" filter on "Personen" page -> we can't select any agb or apb, same for "Gemeente" filter on "Bestuurseenheden"

nbittich commented:
> First off, congrats, it was a big chunk once again grin
> 
>     * When logging in via mock login, we can chose a agb/apb version, but from the frontend code I'd suggest that it's not relevant anymore, can we remove these mock-login accounts ?
> 
>     * I don't see the new fields mentionned in the FP (Naam volgens KBO, verkorte naam, naam volgens statuten, oprichtingsdatum)
> 
>     * I only one type of site, sems to be mising "correspondentieadres", and the site type doesn't seem to be set by default to "maatschappelijke zetel" when creating a new bestuur
> 
>     * "Organisatie" filter on "Personen" page -> we can't select any agb or apb, same for "Gemeente" filter on "Bestuurseenheden"

thanks ! :)

>   When logging in via mock login, we can chose a agb/apb version, but from the frontend code I'd suggest that it's not relevant anymore, can we remove these mock-login accounts ? 

Fixed , deleted the migration

>     * I don't see the new fields mentionned in the FP (Naam volgens KBO, verkorte naam, naam volgens statuten, oprichtingsdatum)

it has been post poned to gs 25 I think (see OP-2266)

>     * I only one type of site, sems to be mising "correspondentieadres", and the site type doesn't seem to be set by default to "maatschappelijke zetel" when creating a new bestuur

I fixed it. please note it's the same when you create a new district, but i didn't fix it for district (new migration also to add correspondantieaddres)

>     * "Organisatie" filter on "Personen" page -> we can't select any agb or apb, same for "Gemeente" filter on "Bestuurseenheden"

there is no mandate for agb apb for now I think.
